src/test/java/ca/team2706/frc/robot/commands/drivebase/DriverAssistVisionTest.java

```diff
@@ -0,0 +1,130 @@
+package ca.team2706.frc.robot.commands.drivebase;
+
+import ca.team2706.frc.robot.config.Config;
+import ca.team2706.frc.robot.subsystems.DriveBase;
+
+import jaci.pathfinder.Pathfinder;
+import jaci.pathfinder.Trajectory;
+
+import mockit.Expectations;
+import mockit.Mocked;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class DriverAssistVisionTest {
+
+    private final DriverAssistVision driverAssistVision = new DriverAssistVision();
+
+    // Since these two classes are accessed statically (they are singletons), we have to mock 
+    // all instances and stub out the static initializer
+
+    @Mocked(stubOutClassInitialization = true)
+    private DriveBase driveBase;
```

Comment:
We generally don't want to mock our own classes, as this could detriment from the realism involved in the tests. 
It's very annoying but we usually just mock all of DriveBase's dependencies themselves rather than DriveBase itself.

Reply:
DriveBase is no longer mocked in DriverAssistVisionTest.java. Mocked classes are similar to what is in DriveBaseTest. Expectations are used on pigeon.getYawPitchRoll().